README.md

```diff
@@ -74,6 +74,18 @@ Metrics Server | Metrics API group/version | Supported Kubernetes version
 
 [Metrics Server releases]: https://github.com/kubernetes-sigs/metrics-server/releases
 
+### High Availability
+
+Latest Metrics Server release can be installed in high availability mode by running:
+
+```shell
+kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/high-availability.yaml
```

Comment:
We should expand our e2e tests to cover both configurations.

Reply:
Good point :+1:. What do you think about adding a new `test-e2e-ha` job for that?

Reply:
Sounds good

Reply:
I also think it is necessary to add tests.
But what is the scope of the tests?  
All our e2e tests to cover both configurations? or do we need to add some alone tests for ha configuration?
For example, We can add the following tests:
- returns accurate CPU and memory metric
- make sure both instances are doing job
- When one replica of Metrics Serve stops, should  return accurate CPU and memory metric, make sure not availability problems

Reply:
For now, I think running the current e2e tests on the HA configuration should be good enough, but in the future, we might want to look into adding tests that are specific to high availability. My only concern would be that we may end up spending time testing Kubernetes features, but it should help to build confidence in the HA configuration.

Reply:
Any concerns with doing the work in a separate PR?

Reply:
Sounds good